### PR TITLE
improve IIS printing

### PR DESCRIPTION
**Pull request by jd-lara**

### CODE REVIEW

**Comment by jd-lara** [comment]
```
 Error: Constraints participating in conflict basis (IIS) 
│ 
│  ┌───────────────────────────────────────────────────────────────┐
│ │ ActivePowerVariableLimitsConstraint__HydroEnergyReservoir__lb │
│ ├───────────────────────────────────────────────────────────────┤
│ │                                       ("BalsaInferior_1", 13) │
│ │                                        ("MiguelDengo_U3", 13) │
│ │                                             ("Chucas_U2", 13) │
│ │                                                ("TAC-U1", 13) │
│ │                                          ("ElGeneral_U1", 13) │
│ │                                             ("Arenal_U2", 13) │
│ │                                             ("Garita_U1", 13) │
│ │                                        ("MiguelDengo_U1", 13) │
│ │                                          ("Sandillal_U2", 13) │
│ │                                              ("NEGII-G1", 13) │
│ │                                            ("ElAngel_U2", 13) │
│ │                                              ("NEGII-G2", 13) │
│ │                                           ("ElAngelA_U2", 13) │
│ │                                           ("DonPedro_U1", 13) │
│ │                                       ("BalsaInferior_2", 13) │
│ │                                             ("Volcan_U1", 13) │
│ │                                        ("MiguelDengo_U2", 13) │
│ │                                             ("Arenal_U3", 13) │
│ │                                       ("BalsaInferior_3", 13) │
│ │                                           ("ElAngelA_U1", 13) │
│ │                                             ("Garita_U2", 13) │
│ │                                          ("ElGeneral_U2", 13) │
│ │                                             ("Arenal_U1", 13) │
│ │                                          ("Sandillal_U1", 13) │
│ │                                             ("Chucas_U1", 13) │
│ │                                                ("TAC-U2", 13) │
│ │                                            ("ElAngel_U1", 13) │
│ └───────────────────────────────────────────────────────────────┘
│ ┌──────────────────────────────────────────────────────────┐
│ │ ActivePowerVariableLimitsConstraint__ThermalStandard__ub │
│ ├──────────────────────────────────────────────────────────┤
│ │                                         ("Moin2_U7", 13) │
│ │                                         ("Moin2_U6", 13) │
│ │                                         ("Moin2_U8", 13) │
│ └──────────────────────────────────────────────────────────┘
│ ┌─────────────────────────────────────────────────┐
│ │           NetworkFlowConstraint__TapTransformer │
│ ├─────────────────────────────────────────────────┤
│ │ ("LIN230A-starbus_53050_53058_53080_1-i_1", 13) │
│ └─────────────────────────────────────────────────┘
│ ┌──────────────────────────────────────────────────────────┐
│ │ ActivePowerVariableLimitsConstraint__ThermalStandard__lb │
│ ├──────────────────────────────────────────────────────────┤
│ │                                      ("Garabito_U3", 13) │
│ │                                      ("Garabito_U5", 13) │
│ │                                   ("Miravalles1_U1", 13) │
│ │                                        ("Pailas2_1", 13) │
│ │                                      ("Garabito_U9", 13) │
│ │                                        ("Pailas_U1", 13) │
│ │                                     ("Garabito_U12", 13) │
│ │                                        ("Pailas_U2", 13) │
│ │                                     ("Garabito_U13", 13) │
│ │                                      ("Garabito_U1", 13) │
│ │                                      ("Garabito_U4", 13) │
│ │                                      ("Garabito_U7", 13) │
│ │                                     ("Garabito_U11", 13) │
│ │                                      ("Garabito_U6", 13) │
│ │                                      ("Garabito_U8", 13) │
│ │                                     ("Garabito_U10", 13) │
│ │                                   ("Miravalles3_U1", 13) │
│ │                                      ("Garabito_U2", 13) │
│ │                                   ("Miravalles2_U1", 13) │
│ └──────────────────────────────────────────────────────────┘
│ ┌─────────────────────────────────────────────────┐
│ │         RateLimitConstraint__TapTransformer__lb │
│ ├─────────────────────────────────────────────────┤
│ │ ("LIN230A-starbus_53050_53058_53080_1-i_1", 13) │
│ └─────────────────────────────────────────────────┘
│ ┌───────────────────────────────────────┐
│ │ CommitmentConstraint__ThermalStandard │
│ ├───────────────────────────────────────┤
│ │                      ("Moin2_U7", 14) │
│ │                      ("Moin2_U6", 14) │
│ │                      ("Moin2_U8", 14) │
│ └───────────────────────────────────────┘
│ ┌──────────────────────────────────────────────────────────────────────┐
│ │ ActivePowerVariableTimeSeriesLimitsConstraint__RenewableDispatch__ub │
│ ├──────────────────────────────────────────────────────────────────────┤
│ │                                                    ("LosSantos", 13) │
│ │                                                 ("ValleCentral", 13) │
│ └──────────────────────────────────────────────────────────────────────┘
│ ┌─────────────────────────────────────────────────────────────────────────┐
│ │ ActivePowerVariableTimeSeriesLimitsConstraint__HydroEnergyReservoir__ub │
│ ├─────────────────────────────────────────────────────────────────────────┤
│ │                                                       ("LaJoya_U2", 13) │
│ │                                                        ("Cachi_U3", 13) │
│ │                                                        ("Lajas_U2", 13) │
│ │                                                     ("RioMacho_U5", 13) │
│ │                                                        ("Cachi_U2", 13) │
│ │                                                     ("RioMacho_U3", 13) │
│ │                                                     ("RioMacho_U4", 13) │
│ │                                                   ("Reventazon_U3", 13) │
│ │                                                   ("Reventazon_U2", 13) │
│ │                                                       ("Torito_U2", 13) │
│ │                                                       ("LaJoya_U1", 13) │
│ │                                                       ("Pirris_U1", 13) │
│ │                                                     ("RioMacho_U1", 13) │
│ │                                                    ("Angostura_U3", 13) │
│ │                                                       ("Pirris_U2", 13) │
│ │                                                        ("Cachi_U1", 13) │
│ │                                                        ("Cachi_U4", 13) │
│ │                                                   ("Reventazon_U5", 13) │
│ │                                                   ("Reventazon_U4", 13) │
│ │                                                     ("RioMacho_U2", 13) │
│ │                                                        ("Lajas_U1", 13) │
│ │                                                       ("LaJoya_U3", 13) │
│ │                                                       ("Torito_U1", 13) │
│ │                                                    ("Angostura_U2", 13) │
│ │                                                   ("Reventazon_U1", 13) │
│ └─────────────────────────────────────────────────────────────────────────┘
│ ┌──────────────────────────────────────┐
│ │ CopperPlateBalanceConstraint__System │
│ ├──────────────────────────────────────┤
│ │                          (50000, 13) │
│ └──────────────────────────────────────┘
```

File: src/core/optimization_container.jl [hunk 1]
```diff
@@ -741,7 +741,13 @@ function compute_conflict!(container::OptimizationContainer)
...
             end
         end
-        @error "$(conflict)"
+
+        msg = IOBuffer()
+        for (k, v) in conflict
+            PrettyTables.pretty_table(msg, v; header = [k])
+        end
+
+        @error "Constraints participating in conflict basis (IIS) \n\n $(String(take!(msg)))"
```
Comment (GabrielKS) [inline src/core/optimization_container.jl:750]:
```suggestion
        @error "Constraints participating in conflict basis (IIS) \n\n$(String(take!(msg)))"
```
fixes the alignment glitch shown in the above comment

**Review by GabrielKS — CHANGES_REQUESTED** [review]
Looks good to me aside for the one typo

File: src/core/optimization_container.jl (hunk 1 above)
Comment (jd-lara) [reply src/core/optimization_container.jl]: done

**Review by GabrielKS — APPROVED** [review]

**Pull request merged**